CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 3.15
-* Update JS/TS/CSS analyzer 9.13.0 -> [10.0.0](https://github.com/SonarSource/SonarJS/releases/tag/10.0.0.20728), support for JavaScript analysis inside HTML files, FN and FP fixes, dependency upgrades
+* Update JS/TS/CSS analyzer 9.13.0 -> [10.0.1](https://github.com/SonarSource/SonarJS/releases/tag/10.0.1.20755), support for JavaScript analysis inside HTML files, FN and FP fixes, dependency upgrades
+* Update CFamily analyzer 6.41.0 -> [6.42.0](https://sonarsource.atlassian.net/issues/?jql=project%20%3D%2010166%20AND%20fixVersion%20%3D%2013995%20ORDER%20BY%20priority%20DESC%2C%20key%20ASC), Support for clang-cl and Microchip compilers
+* Update Python analyzer 3.21 -> [4.0.0](https://github.com/SonarSource/sonar-python/releases/tag/4.0.0.11155), New quick fixes available, FN and FP fixes
```

Comment:
I believe the intermediate version(s) could be mentioned here, with a link to the related changelog(s).

Reply:
Should be resolved by the latest change [601658c](https://github.com/SonarSource/sonarlint-vscode/pull/349/commits/601658c7d8e9fd96a9197559dab2719b5cf22bbb)